src/connections/connections.controller.ts

```diff
@@ -0,0 +1,97 @@
+import { Controller, Delete, Get, Param, Post } from '@nestjs/common';
+import { CurrentUser } from 'src/decorators/current-user.decorator';
+import { ConnectionsService } from './connections.service';
+import { ConnectionDto } from './DTO/Connection.dto';
+import {
+  ApiBadRequestResponse,
+  ApiBearerAuth,
+  ApiNotFoundResponse,
+} from '@nestjs/swagger';
+import { User } from '@prisma/client';
+import { Public } from 'src/decorators/public.decorator';
+
+@ApiBearerAuth()
+@Controller('connections')
```

Comment:
Our existing endpoints use singular in the controller name, so we should be sticking to `connection` in here.

Reply:
This is a larger discussions. I think it makes sense to have `user` since in that controller we're usually dealing with a single user. 
Put in `reviews` and `connections` we are usually dealing with multiple resources. See: [Google's api design guidelines](https://cloud.google.com/apis/design/resource_names)
and 
[this StackOverflow discussion](https://stackoverflow.com/questions/6845772/should-i-use-singular-or-plural-name-convention-for-rest-resources)

Reply:
This makes sense